monitor/census.go

```diff
@@ -1842,7 +1842,7 @@ func AIRequestError(code string, Pipeline string, Model string, orchInfo *lpnet.
 	}
 
 	if err := stats.RecordWithTags(census.ctx,
-		[]tag.Mutator{tag.Insert(census.kErrorCode, code), tag.Insert(census.kPipeline, Pipeline), tag.Insert(census.kModelName, Model), tag.Insert(census.kOrchestratorURI, orchInfo.GetTranscoder()), tag.Insert(census.kOrchestratorAddress, orchAddr)},
+		[]tag.Mutator{tag.Insert(census.kErrorCode, code), tag.Insert(census.kPipeline, Pipeline), tag.Insert(census.kModelName, Model), tag.Insert(census.kOrchestratorURI, orchInfo.GetTranscoder()), tag.Insert(census.kOrchestratorAddress, orchAddr), tag.Insert(census.kOrchestratorVersion, orchInfo.GetCapabilities().GetVersion())},
```

Comment:
I'm don't know what tag/label would be added if `orchInfo.GeCapabilities()` returns nil and why would it return `nil`. Max [implemented it differently](https://github.com/livepeer/go-livepeer/pull/3158/files#diff-ee88404ba626f86b6e9a0b24206faba9622ea02ef5afd2044ae3b4e06938836dR920-R921)